yt_dlp/extractor/eggs.py

```diff
@@ -0,0 +1,155 @@
+import secrets
```

Comment:
I don't think we use this module anywhere else in yt-dlp. In the interest of saving on executable filesize, could we do something like this?

```diff
     def _real_initialize(self):
-        self._API_HEADERS['deviceId'] = secrets.token_hex(8)
+        self._API_HEADERS['deviceId'] = ''.join(random.choices(string.hexdigits[:16], k=16))
```

Reply:
We could. Though would have suggested going the other way and replacing some `''.join(random.choices` instances with `secrets.token...` calls.